### ISSUE TITLE
Burp Customizer Extension is not working

### ISSUE THREAD

**Issue by SecureNeo** [CoreyD97/BurpCustomizer#16]
I have installed the Customizer extension from BApp Store without any errors, but the theme selected is not reflecting to the application. What would I be missing?  or any recent changes made in code?

![image](https://user-images.githubusercontent.com/74350151/142137962-982ea73e-5f3c-4e22-8f59-2a78d780feef.png)

**Comment by CoreyD97** [comment]
There's an apply button at the bottom that doesn't seem to be visible in your screenshot due to screen resolution issues. This is fixed in a later version but it hasn't been pushed to the BApp store yet. Try downloading the latest version on the Github release page.

I've submitted a request for the version on the BApp store to be updated.

**Comment by Rehman-AEN-Github** [comment]
Yes, there's a problem with my VDI screen resolution managed by an administrator. However, the new release v1.0.3 fixed the issue. Thanks @CoreyD97 🙂
